README.md

```diff
@@ -14,157 +14,210 @@
 
 ## What is Fundamental Library Styles?
 
-Fundamental Library Styles is a light-weight presentation layer that can be used with your UI framework of choice (e.g. Angular, React, Vue, etc.). With Fundamental Library Styles, library of stylesheets and HTML tags, developers can build consistent Fiori apps in any web-based technology.
+Fundamental Library Styles is a light-weight presentation layer that can be used with your UI framework of choice (e.g.
+Angular, React, Vue, etc.). With Fundamental Library Styles, library of stylesheets and HTML tags, developers can build
+consistent Fiori apps in any web-based technology.
```

Comment:
Fundamental Library Styles is a light-weight presentation layer that can be used with **developers'** UI framework of choice (e.g.Angular, React, Vue, etc.) 

I suggest developers' in order to stay consistent with the next sentence. 

As per the second sentence, I would suggest avoiding starting the sentence with "With".  Maybe the sentence can be rewritten:
Developers can build consistent Fiori apps in any web-based technology, by combining Fundamental Styles, library of stylesheets and HTML tags.